File: GSI QA testing tool/UI/UI_014_IndividualRegistration.cs
using GSI_QA_testing_tool.Utilities;
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GSI_QA_testing_tool.UI
{
    internal class UI_014_IndividualRegistration
    {
        public static void GoTo(IWebDriver driver)
        {


            if (Data._StopAt.Contains("Individual Registration Type"))
            {
                CustomDialog customDialog = new CustomDialog("Make Selection", "Select pages you want to stop at, make changes and click ok before clicking the Next button");
                customDialog.ShowDialog();
            }





            if (Finder.FindIt(driver, "//a[@id='ctl00_Main_content_lnkCompleteHeader']")) 
            {
                Debug.WriteLine("IndividualRegistration is On");
            }
            else
            {
                Debug.WriteLine("IndividualRegistration is Off");
                return;
            }



            Finder.ClickIt(driver, "//a[@id='ctl00_Main_content_lnkCompleteHeader']");
        }
    }
}

File: GSI QA testing tool/UI/UI_014_IndividualRegistration.vb
﻿Imports OpenQA.Selenium
Imports System
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Linq
Imports System.Text
Imports System.Threading
Imports System.Threading.Tasks

Namespace GSI_QA_testing_tool.UI
    Friend Class UI_014_IndividualRegistration
        Public Shared Sub [GoTo](ByVal driver As IWebDriver, ByVal js As IJavaScriptExecutor)

            Thread.Sleep(3000)


            If Data._StopAt.Contains("Individual Registration Type") Then
                Data.StopAtDialog()
            End If

            If Finder.FindItByCSS(driver, "[id='ctl00_Main_content_lnkCompleteHeader']") Then
                Debug.WriteLine("IndividualRegistration is On")
            Else
                Debug.WriteLine("IndividualRegistration is Off")
                Return
            End If

            Finder.JSClickIt(driver, "#ctl00_Main_content_lnkCompleteHeader", js)
        End Sub
    End Class
End Namespace
